src/components/PortfolioAccordions.vue

```diff
@@ -87,7 +92,7 @@
           aria-expanded="true"
           aria-controls="m-a3"
         >
-          Events
+          Events / Current Conditions
```

Comment:
I don't think we need to change this now, but I think it might be good to have river conditions in a separate section from events